configure.ac

```diff
@@ -652,6 +652,13 @@ AX_PROG_BISON
 dnl AC_PATH_PROG([BISON],[bison])
 dnl AX_PROG_BISON_VERSION([3.0.2],[],[ ... ])
 
+AC_ARG_VAR(DIFF, diff command)
```

Comment:
This sets the variable as precious, which is good; but shouldn't the program itself be configured with a "generic program" macro?
https://www.gnu.org/software/autoconf/manual/autoconf-2.70/html_node/Generic-Programs.html

Reply:
Wouldn't `AC_PATH_PROG` be sufficient (as for `bison` above) ?

Reply:
It would suffice for infrastructure and the option to override on the command line - but it wouldn't have any checks on that and always default to the first one.
Using `AC_PATH_PROGS_FEATURE_CHECK` would allow to additionally check for the exact thing we want to have working and would test programs "later in PATH" (like the one you've manually used for the override) for that - so a manual override won't be necessary any more.